proto/toit/api/organization.proto

```diff
@@ -109,3 +110,12 @@ message UpdateOrganizationRequest {
 }
 
 message UpdateOrganizationResponse {}
+
+message CreateNewOrganizationRequest {
+  string name = 1;
+  string email = 2;
+}
+
+message CreateNewOrganizationResponse {
+
```

Comment:
I still think we should return the organization id.

Reply:
ah of course, I forgot that